keyboards/keycapsss/kimiko/rev2/keymaps/default/keymap.c

```diff
@@ -0,0 +1,129 @@
+    KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                                           KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    _______,
+    KC_F1,  KC_F2,    KC_F3,   KC_F4,   KC_F5,   KC_F6,                                          XXXXXXX, KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT, XXXXXXX,
+    KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,   _______, _______,  _______, _______,  KC_PLUS, KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+                               _______, _______, _______,  _______, _______,  _______, _______,  _______, _______, _______
+),
+/* ADJUST (Press LOWER and RAISE together)
+ * ,--------------------------------------------.                                   ,----------------------------------------------.
+ * | QK_BOOT |      |      |      |     |       |                                   |      |      |      |      |      |           |
+ * |---------+------+------+------+------+------|                                   |------+------+------+------+------+-----------|
+ * | RGB ON  | HUE+ | SAT+ | VAL+ |      |      |                                   | PREV | PLAY | NEXT |      |      |           |
+ * |---------+------+------+------+------+------|                                   |------+------+------+------+------+-----------|
+ * | MODE    | HUE- | SAT- | VAL- |      |      |---------------.   ,---------------| VOL+ | MUTE | VOL- |      |      |           |
+ * |---------+------+------+------+------+------|      |        |   |       |       |------+------+------+------+------+-----------|
+ * |         |      |      |      |      |      |------|--------|   |-------|-------|      |      |      |      |       |           |
+ * `--------------------------------------------|               /    \              |----------------------------------------------'
+ *                         | LCTRL| LGUI |LALT  |LOWER | Space /      \Enter| RAISE |BackSP| RGUI | RALT |
+ *                         `----------------------------------'        '---------------------------------'
+ */
+[_ADJUST] = LAYOUT(
+    QK_BOOT,   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+    RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, XXXXXXX, XXXXXXX,                                     KC_MPRV, KC_MPLY, KC_MNXT, XXXXXXX, XXXXXXX, XXXXXXX,
+    RGB_MOD, RGB_HUD, RGB_SAD, RGB_VAD, XXXXXXX, XXXXXXX,                                     KC_VOLU, KC_MUTE, KC_VOLD, XXXXXXX, XXXXXXX, XXXXXXX,
+    XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+                               _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______
+)
+};
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+    state = update_tri_layer_state(state, _RAISE, _LOWER, _ADJUST);
+    return state;
+}
```

Comment:
Tri layer feature should be used and enabled here instead:
https://docs.qmk.fm/#/feature_tri_layer

Eg, `TL_LOWR` and `TL_UPPR` perform the MO functionality, plus the adjust layer handling.

Reply:
Thanks for the hint.
Included this feature.

Reply:
You still need to remove this block
```suggestion
```